### DISCUSSION TITLE
I cannot import createServerSideHelpers function

### DISCUSSION THREAD

**Discussion by dogukanoksuz** [trpc/trpc#4202] [General]
Hey,

I met tRPC with Create-T3-App stack and try to build an application with it.

I wanna server side render my pages but i cannot import createServerSideHelpers from mentioned '@trpc/react-query/server' package.

Am I doing something wrong? Even i looked at node_modules folder and this function does not exists here.

```
  '@trpc/client':
    specifier: ^10.18.0
    version: 10.18.0(@trpc/server@10.18.0)
  '@trpc/next':
    specifier: ^10.18.0
    version: 10.18.0(@tanstack/react-query@4.28.0)(@trpc/client@10.18.0)(@trpc/react-query@10.18.0)(@trpc/server@10.18.0)(next@13.3.0)(react-dom@18.2.0)(react@18.2.0)
  '@trpc/react-query':
    specifier: ^10.18.0
    version: 10.18.0(@tanstack/react-query@4.28.0)(@trpc/client@10.18.0)(@trpc/server@10.18.0)(react-dom@18.2.0)(react@18.2.0)
  '@trpc/server':
    specifier: ^10.18.0
    version: 10.18.0
```

**Comment by sachinraja** [comment]
update to the latest version of tRPC

**Reply by KATT** [reply]
☝️ 

```sh
npx npm-check-updates /@trpc/ -u
```

**Comment by dogukanoksuz** [comment]
Thanks, now i can import but pattern was not like i was imagined.

https://github.com/dogukanoksuz/dogukan.dev/blob/2fc00579e8661b5986f78c9b7d907169618ce238/src/pages/%5Bslug%5D.tsx#L15

Using SSR: true like this is an anti-pattern usage?

Thanks for helping.
